python/test/unit/la/test_matrix_csr.py

```diff
@@ -215,3 +215,22 @@ def test_set_diagonal_distributed(dtype):
     A.finalize()
     assert (As.diagonal()[nlocal:] == dtype(0.)).all()
     assert (As.diagonal()[:nlocal] == dtype(1.)).all()
+
+
+@pytest.mark.parametrize('dtype', [np.float32, np.float64, np.complex64, np.complex128])
+def test_bad_entry(dtype):
+    sp = create_test_sparsity(6, 1)
+    mat1 = matrix_csr(sp, dtype=dtype)
+
+    # Set block in bs=1 matrix
+    with pytest.raises(RuntimeError):
+        mat1.set([1.0, 2.0, 3.0, 4.0], [0], [0], 2)
+    # Normal
```

Comment:
What does this comment mean?

Reply:
Data has block size but matrix does not (bs=1)

Reply:
My comment was in invitation to improve the code comment :). `# Normal` carries no meaning.

Reply:
Sure will do.